### ISSUE TITLE
Weird things randomly happened this morning

### ISSUE THREAD

**Issue by jscottelblein** [causefx/Organizr#1605]
###### Organizr Version: V 1.x
v2 (latest  v2 dev container as of last night)
###### Branch: Master/Develop
Develop
###### WebServer: Nginx/Apache
Nginx
###### Operating System: Windows/MacOS/Ubuntu
Linux Docker
<hr>

##### Problem Description:

Some random things happened that I noticed the first time login in this morning:

- 2FA keeps saying it's wrong, even though it's been fine for a while now and I didn't change anything. I even tried re-enabling it and using the new hash it gives and it still says wrong 2fa. I have to use the Plex oath login now in order  to get in.
- All of the Tautulli items in the homepage widget don't load; only 2 of them.
- I've had the Organizr settings icon moved to a submenu item for a long time; that settings icon just up and vanished; now I can't get to the settings page (I forgot the direct URL).
- Organizr also loads real slow; it was running at normal speed when I went to be last night.
- the qBittorrent icon disappeared from the consolidated downloader widget on the homepage. I've confirmed that qBittorrent itself is still running as usual.

So  far that's all I've noticed, but may not be all inclusive. I've already done the usual basic checks; restarting docker, restarting the PC, etc.

Any ideas?

<hr>

##### Reproduction Steps:

No idea.
<hr>

#### Errors on screen?  If so paste here:

```

```

**Comment by tronyx** [comment]
Sounds like issues with the underlying machine, not Organizr itself. Do the missing icons still actually exist within the local filesystem? If so, are they still accessible via a browser?

**Comment by Roxedus** [comment]
Post some container logs. and what architecture are you on?

**Comment by jscottelblein** [comment]
> Sounds like issues with the underlying machine, not Organizr itself. Do the missing icons still actually exist within the local filesystem? If so, are they still accessible via a browser?

@tronyx The icons (settings cog wheel, and qBittorrent icon) are the standard icons that come with Organizr. The other standard icons that come with it are all loading up fine. Not sure of their exact paths in the container right now, to try directly loading them.

**Comment by jscottelblein** [comment]
> Post some container logs. and what architecture are you on?

@Roxedus I'm using the latest official v2 dev container, running in docker desktop on win 10 WSL2. I'm not seeing anything unusual in the container log:

```
2021-03-05T22:18:28.208013700Z [s6-init] making user provided files available at /var/run/s6/etc...exited 0.
2021-03-05T22:18:28.841744400Z [s6-init] ensuring user provided files have correct perms...exited 0.
2021-03-05T22:18:28.851176300Z [fix-attrs.d] applying ownership & permissions fixes...
2021-03-05T22:18:28.886790100Z [fix-attrs.d] done.
2021-03-05T22:18:28.909427200Z [cont-init.d] executing container initialization scripts...
2021-03-05T22:18:28.928476000Z [cont-init.d] 10-adduser: executing... 
2021-03-05T22:18:37.660377900Z 
2021-03-05T22:18:37.660407200Z -------------------------------------
2021-03-05T22:18:37.660413600Z GID/UID
2021-03-05T22:18:37.660419900Z -------------------------------------
2021-03-05T22:18:37.660425500Z User uid:    1000
2021-03-05T22:18:37.660431100Z User gid:    1000
2021-03-05T22:18:37.660436700Z -------------------------------------
2021-03-05T22:18:37.660442600Z 
2021-03-05T22:18:37.668502900Z [cont-init.d] 10-adduser: exited 0.
2021-03-05T22:18:37.669998200Z [cont-init.d] 20-config: executing... 
2021-03-05T22:18:37.861639400Z [cont-init.d] 20-config: exited 0.
2021-03-05T22:18:37.863587200Z [cont-init.d] 30-setup: executing... 
2021-03-05T22:18:38.839673900Z [cont-init.d] 30-setup: exited 0.
2021-03-05T22:18:38.842150300Z [cont-init.d] 40-install: executing... 
2021-03-05T22:18:38.852515900Z -----------------------
2021-03-05T22:18:38.852545700Z |  Updating Organizr  |
2021-03-05T22:18:38.852552100Z -----------------------
2021-03-05T22:18:44.029849200Z Updating files:   4% (218/4924)
Updating files:   5% (247/4924)
Updating files:   6% (296/4924)
Updating files:   7% (345/4924)
Updating files:   8% (394/4924)
Updating files:   9% (444/4924)
Updating files:  10% (493/4924)
Updating files:  11% (542/4924)
Updating files:  12% (591/4924)
Updating files:  13% (641/4924)
Updating files:  14% (690/4924)
Updating files:  15% (739/4924)
Updating files:  16% (788/4924)
Updating files:  17% (838/4924)
Updating files:  18% (887/4924)
Updating files:  19% (936/4924)
Updating files:  20% (985/4924)
Updating files:  21% (1035/4924)
Updating files:  22% (1084/4924)
Updating files:  23% (1133/4924)
Updating files:  24% (1182/4924)
Updating files:  25% (1231/4924)
Updating files:  26% (1281/4924)
Updating files:  27% (1330/4924)
Updating files:  28% (1379/4924)
Updating files:  29% (1428/4924)
Updating files:  30% (1478/4924)
Updating files:  31% (1527/4924)
Updating files:  32% (1576/4924)
Updating files:  33% (1625/4924)
Updating files:  34% (1675/4924)
Updating files:  35% (1724/4924)
Updating files:  36% (1773/4924)
Updating files:  37% (1822/4924)
Updating files:  37% (1838/4924)
Updating files:  38% (1872/4924)
Updating files:  39% (1921/4924)
Updating files:  40% (1970/4924)
Updating files:  41% (2019/4924)
Updating files:  42% (2069/4924)
Updating files:  43% (2118/4924)
Updating files:  44% (2167/4924)
Updating files:  45% (2216/4924)
Updating files:  46% (2266/4924)
Updating files:  47% (2315/4924)
Updating files:  48% (2364/4924)
Updating files:  49% (2413/4924)
Updating files:  50% (2462/4924)
Updating files:  51% (2512/4924)
Updating files:  52% (2561/4924)
Updating files:  53% (2610/4924)
Updating files:  54% (2659/4924)
Updating files:  55% (2709/4924)
Updating files:  56% (2758/4924)
Updating files:  57% (2807/4924)
Updating files:  58% (2856/4924)
Updating files:  59% (2906/4924)
Updating files:  60% (2955/4924)
Updating files:  61% (3004/4924)
Updating files:  62% (3053/4924)
Updating files:  63% (3103/4924)
Updating files:  64% (3152/4924)
Updating files:  65% (3201/4924)
Updating files:  66% (3250/4924)
Updating files:  67% (3300/4924)
Updating files:  67% (3306/4924)
Updating files:  68% (3349/4924)
Updating files:  69% (3398/4924)
Updating files:  70% (3447/4924)
Updating files:  71% (3497/4924)
Updating files:  72% (3546/4924)
Updating files:  73% (3595/4924)
Updating files:  74% (3644/4924)
Updating files:  75% (3693/4924)
Updating files:  76% (3743/4924)
Updating files:  77% (3792/4924)
Updating files:  78% (3841/4924)
Updating files:  79% (3890/4924)
Updating files:  80% (3940/4924)
Updating files:  81% (3989/4924)
Updating files:  82% (4038/4924)
Updating files:  83% (4087/4924)
Updating files:  84% (4137/4924)
Updating files:  85% (4186/4924)
Updating files:  86% (4235/4924)
Updating files:  87% (4284/4924)
Updating files:  88% (4334/4924)
Updating files:  89% (4383/4924)
Updating files:  90% (4432/4924)
Updating files:  91% (4481/4924)
Updating files:  92% (4531/4924)
Updating files:  93% (4580/4924)
Updating files:  94% (4629/4924)
Updating files:  95% (4678/4924)
Updating files:  96% (4728/4924)
Updating files:  97% (4777/4924)
Updating files:  98% (4826/4924)
Updating files:  99% (4875/4924)
Updating files: 100% (4924/4924)
Updating files: 100% (4924/4924), done.
2021-03-05T22:18:44.200123800Z HEAD is now at f00da8a0 Merge pull request #1601 from causefx/v2-develop
2021-03-05T22:18:44.332611600Z Already on 'v2-master'
2021-03-05T22:18:44.332918100Z Your branch is up to date with 'origin/v2-master'.
2021-03-05T22:18:44.408088800Z HEAD is now at f00da8a0 Merge pull request #1601 from causefx/v2-develop
2021-03-05T22:18:44.422469500Z hint: Pulling without specifying how to reconcile divergent branches is
2021-03-05T22:18:44.422540400Z hint: discouraged. You can squelch this message by running one of the following
2021-03-05T22:18:44.422547600Z hint: commands sometime before your next pull:
2021-03-05T22:18:44.422553500Z hint: 
2021-03-05T22:18:44.422559200Z hint:   git config pull.rebase false  # merge (the default strategy)
2021-03-05T22:18:44.422564800Z hint:   git config pull.rebase true   # rebase
2021-03-05T22:18:44.422570400Z hint:   git config pull.ff only       # fast-forward only
2021-03-05T22:18:44.422610000Z hint: 
2021-03-05T22:18:44.422647200Z hint: You can replace "git config" with "git config --global" to set a default
2021-03-05T22:18:44.422653800Z hint: preference for all repositories. You can also pass --rebase, --no-rebase,
2021-03-05T22:18:44.422659800Z hint: or --ff-only on the command line to override the configured default per
2021-03-05T22:18:44.422665500Z hint: invocation.
2021-03-05T22:18:44.906687300Z Already up to date.
2021-03-05T22:18:45.043941000Z [cont-init.d] 40-install: exited 0.
2021-03-05T22:18:45.047031500Z [cont-init.d] 50-debug: executing... 
2021-03-05T22:18:45.069764900Z -----------------------
2021-03-05T22:18:45.069800900Z |     Kernel Info     |
2021-03-05T22:18:45.069823600Z -----------------------
2021-03-05T22:18:45.071057300Z Linux organizr 5.4.72-microsoft-standard-WSL2 #1 SMP Wed Oct 28 23:40:43 UTC 2020 x86_64 GNU/Linux
2021-03-05T22:18:45.071241200Z Time Zone: America/Chicago
2021-03-05T22:18:45.072891600Z [cont-init.d] 50-debug: exited 0.
2021-03-05T22:18:45.078032100Z [cont-init.d] done.
2021-03-05T22:18:45.100566900Z [services.d] starting services
2021-03-05T22:18:45.277342700Z [services.d] done.
```

**Comment by causefx** [comment]
the 2fa issue is more than likely caused by incorrect server time.

**Comment by jscottelblein** [comment]
> the 2fa issue is more than likely caused by incorrect server time.

@causefx Weird; at the WSL/Ubuntu prompt I do a `date` and  it matches the same time as on the windows host itself, inside the  Organizr container, and the machine I try to use the 2fa code from.

the 2 things that weird me out the most are the Organizr icons vanishing, especially the settings icon.

**Comment by causefx** [comment]
have you checked nginx logs as well?

**Comment by tronyx** [comment]
You're going to close it without commenting? Did you resolve the issues?

**Comment by jscottelblein** [comment]
> You're going to close it without commenting? Did you resolve the issues?

Yea. Felt too r-tarded to bother commenting. ;)

Burnout is real! Turns out I had somehow mistakenly logged into a different user (with a very similar name) who only had co-admin perms, so certain things weren't showing up. First time that's ever happened, since I only made the other user as a test, and never log into it. lmao. I'm kinda surprised you guys didn't think about that being a possibility! :-P

As far as the speed, I'm not sure why that was happening yet, but it seems to have fixed itself.
